src/startup_api.py

```
@@ -0,0 +1,291 @@
+        try:
+            load_whisper_model()
+        except Exception as e:
+            logger.warning(f"⚠️ Whisper model failed to load (non-critical): {e}")
+            logger.info("Continuing without Whisper - core emotion/summarization models loaded successfully")
+
+        # Log memory usage after loading
+        try:
+            memory_after = psutil.virtual_memory()
+            logger.info(f"Memory after loading: {memory_after.used / (1024**3):.2f}GB used / {memory_after.total / (1024**3):.2f}GB total")
+            logger.info(f"Memory increase: {(memory_after.used - memory_before.used) / (1024**3):.2f}GB")
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)